mobile/README.md

```diff
@@ -38,6 +61,26 @@ To checkout the latest tagged release of lnd, run
 ⛰  git checkout $(git describe --tags --abbrev=0)
 ```
 
+Or, alternatively, you can clone the project and checkout to any branch:
```

Comment:
Perhaps formulate this a bit differently? Since "alternatively" here means "if you haven't already cloned the project".